reference/docs-conceptual/learn/ps101/10-script-modules.md

```diff
@@ -0,0 +1,380 @@
+unapproved verbs that might make them less discoverable. To find the commands with
+unapproved verbs, run the Import-Module command again with the Verbose parameter. For a
+list of approved verbs, type Get-Verb.
+```
+
+Just to reiterate, although the `New-Module` cmdlet was used in the previous example, that's not the
+command for creating script modules in PowerShell.
+
+Save the following two functions in a file named MyScriptModule.psm1.
+
+```powershell
+function Get-MrPSVersion {
+    $PSVersionTable
+}
+
+function Get-MrComputerName {
+    $env:COMPUTERNAME
+}
+```
+
+Try to call one of the functions.
+
+```powershell
+Get-MrComputerName
+```
+
+```Output
+Get-MrComputerName : The term 'Get-MrComputerName' is not recognized as the name of a
+cmdlet, function, script file, or operable program. Check the spelling of the name, or
+if a path was included, verify that the path is correct and try again.
+At line:1 char:1
++ Get-MrComputerName
+    + CategoryInfo          : ObjectNotFound: (Get-MrComputerName:String) [], CommandNot
+   FoundException
+    + FullyQualifiedErrorId : CommandNotFoundException
+```
+
+An error message is generated saying the function can't be found. You could also check the function
+PSDrive just like before and you'll find that it doesn't exist there either.
+
+You could manually import the file with the `Import-Module` cmdlet.
+
+```powershell
+Import-Module C:\MyScriptModule.psm1
+```
+
+The module autoloading feature was introduced in PowerShell version 3. To take advantage of module
+autoloading, a script module needs to be saved in a folder with the same base name as the `.PSM1` file
+and in a location specified in `$env:PSModulePath`.
+
+```powershell
+$env:PSModulePath
+```
+
+```Output
+C:\Users\mike-ladm\Documents\WindowsPowerShell\Modules;C:\Program Files\WindowsPowerShell\
+Modules;C:\Windows\system32\WindowsPowerShell\v1.0\Modules;C:\Program Files (x86)\Microsof
+t SQL Server\130\Tools\PowerShell\Modules\
+```
+
+The results are difficult to read. Since the paths are separated by a semicolon, you can split the
+results to return each path on a separate line. This makes them easier to read.
+
+```powershell
+$env:PSModulePath -split ';'
+```
+
+```Output
+C:\Users\mike-ladm\Documents\WindowsPowerShell\Modules
+C:\Program Files\WindowsPowerShell\Modules
+C:\Windows\system32\WindowsPowerShell\v1.0\Modules
+C:\Program Files (x86)\Microsoft SQL Server\130\Tools\PowerShell\Modules\
+```
+
+The first three paths in the list are the default. When SQL Server Management Studio was installed,
+it added the last path. For module autoloading to work, the `MyScriptModule.psm1` file needs to be
+located in a folder named `MyScriptModule` directly inside one of those paths.
+
+Not so fast. For me, the current user path is the first one in the list. I almost never use that
+path since I log into Windows as a different user than which I run PowerShell. That means it's not
+located in my normal Documents folder.
```

Comment:
```suggestion
Not so fast. For me, my current user path isn't the first one in the list. I almost never use that
path since I log into Windows as a different user than I run PowerShell. That means it's not
located in my normal Documents folder.
```

Reply:
Fixed